### PR TITLE
Do not requireReturnForObjectLiteral

### PR DESCRIPTION
**Pull request by janfigala**
disable requireReturnForObjectLiteral in arrow functions as consensus thinking is that this rule isn't adding any value and is actually annoying for cases like mapStateToProps in React etc, where you're trying to do something simple and the rule forces you to add extra braces and a return statement.

Only one line was changed here but Prettier kindly formatted the whole file. Let me know if you'd like me to revert

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,13 +7,10 @@ module.exports = {
...
     },
-    plugins: [
-        "react",
-        "import"
-    ],
+    plugins: ["react", "import"],
     rules: {
         "array-bracket-spacing": [2, "never"],
-        "arrow-body-style": [2, "as-needed", { requireReturnForObjectLiteral: true }],
+        "arrow-body-style": [2, "as-needed", { requireReturnForObjectLiteral: false }],
```
Comment (braposo) [inline index.js:13]: I think that's already the default option isn't it? It's fine to leave it explicit like this but you could also remove if you prefer to keep it simple!

**Review by braposo — APPROVED** [review]
@janfigala looks good, just a minor comment.

I think we should put this up for discussion in the next FE meeting because we're using this package as the standard for all of our recent code and it's using `eslint@3.5.0` which is 2 major versions behind the current version `5.2.0`.

We need to update this, include the new eslint recommended rules and make the necessary changes to fit our style guidelines. 

I understand this is a bigger task than this PR but I feel that we'll need to address this sooner rather than later because it'll impact any future code. I've added an issue to the FE tech debt board just to keep track.

File: index.js (hunk 1 above)
Comment (janfigala) [reply index.js:13]: I don't mind either way.

**Comment by janfigala** [comment]
Thanks for adding the above to Tech Debt board @braposo. Definitely needs to be upgraded :D

**Pull request merged**